content/docs/reference-events.md

```diff
@@ -244,33 +245,33 @@ string pointerType
 boolean isPrimary
 ```
 
-A note on cross-browser support:
+Una nota sobre la compatibilidad con varios navegadores:
 
-Pointer events are not yet supported in every browser (at the time of writing this article, supported browsers include: Chrome, Firefox, Edge, and Internet Explorer). React deliberately does not polyfill support for other browsers because a standard-conform polyfill would significantly increase the bundle size of `react-dom`.
+Los eventos de puntero aún no son compatibles con todos los navegadores (al momento de escribir este artículo, los navegadores compatibles incluyen: Chrome, Firefox, Edge e Internet Explorer). Reaccionar de forma deliberada no admite el polietileno para otros navegadores, ya que un polietileno de conformidad estándar aumentaría significativamente el tamaño del paquete de `react-dom`.
```

Comment:
```suggestion
Los eventos de puntero aún no son compatibles con todos los navegadores (en el momento de escritura de este artículo, los navegadores compatibles incluyen: Chrome, Firefox, Edge e Internet Explorer). Reaccionar de forma deliberada no admite el polietileno para otros navegadores, ya que un polyfill de conformidad estándar aumentaría significativamente el tamaño del paquete de `react-dom`.
```

`Polietileno` sounds like a Google Translate translation (it's an chemical organic molecule) and makes no sense in this context.

Reply:
Like I've mentioned [here](https://github.com/reactjs/es.reactjs.org/pull/29#discussion_r253152517), polyfill is a technical word that doesn't really have a translation. I would leave it as is.

Reply:
@hugmanrique Yes this was google translate, my spanish speaking friend didn't have a word for this and I wasn't sure if polyfill would be considered as technical, thanks!